git_machete/git_operations.py

```diff
@@ -198,6 +198,7 @@ def __init__(self) -> None:
         self.__is_equivalent_tree_reachable_cached: Dict[Tuple[FullCommitHash, FullCommitHash], bool] = {}
         self.__local_branches_cached: Optional[List[LocalBranchShortName]] = None
         self.__merge_base_cached: Dict[Tuple[FullCommitHash, FullCommitHash], Optional[FullCommitHash]] = {}
+        self.__missing_tracking_branch: Optional[set[str]] = None
```

Comment:
Nitpick:
```suggestion
        self.__missing_tracking_branch: Optional[Set[str]] = None
```
for consistency (if `mypy` accepts that)

Reply:
ah, that should fix py 3.6-3.8